packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -55,7 +55,7 @@ export abstract class ControllerInterface {
       throw this.errorFactory_.create(ERROR_CODES.BAD_SENDER_ID);
     }
 
-    this.messagingSenderId_ = app.options[SENDER_ID_OPTION_NAME]!;
+    this.messagingSenderId_ = app.options[SENDER_ID_OPTION_NAME];
```

Comment:
so this can be null? It shouldn't right?

Reply:
I had to remove these because --strict (and --strictNullChecks with it) was disabled yesterday. I have another PR coming soon to reenable strict and revert this change.